project/cmake/README.md

```diff
@@ -168,13 +221,36 @@ scanning them would take up an unreasonable amount of configure time.
 When using the makefile builds a few extra targets are defined:
 
 - `make check` builds and executes the test suite.
+- `make check-valgrind` builds and executes the test suite with valgrind memcheck.
 
 Code coverage (with Gcov, LCOV and Gcovr) can be built on Linux:
 
 - CMake has to be executed with `-DCMAKE_BUILD_TYPE=Coverage`
 - `make coverage` generates an HTML code coverage report.
 - `make coverage_xml` generates an XML code coverage report.
 
+## Building binary addons
```

Comment:
see my post on the forum: http://forum.kodi.tv/showthread.php?tid=219166&pid=1934922#pid1934922
developing addons is cumbersome. if the addon does not exist in our binary-addon-repo you have to create a folder manually in project/cmake/addons/addons or the build systems fails silently